api/tests/unit/application/usecases/checkUserCanAccessCampaign_test.js

```diff
@@ -0,0 +1,50 @@
+const { expect, sinon, domainBuilder } = require('../../../test-helper');
+const useCase = require('../../../../lib/application/usecases/checkUserCanAccessCampaign');
+const userRepository = require('../../../../lib/infrastructure/repositories/user-repository');
+const campaignRepository = require('../../../../lib/infrastructure/repositories/campaign-repository');
+
+describe('Unit | Application | Use Case | CheckUserCanAccessCampaign', () => {
+
+  beforeEach(() => {
+    sinon.stub(userRepository, 'getWithMemberships');
+    sinon.stub(campaignRepository, 'get');
+  });
+
+  context('When user can access campaign', () => {
+
+    it('should return true', async () => {
+      // given
+      const organization = domainBuilder.buildOrganization();
+      const membership = domainBuilder.buildMembership({ organization });
+      const user = domainBuilder.buildUser({ memberships: [membership] });
+      const campaign = domainBuilder.buildCampaign({ organizationId: organization.id });
```

Comment:
```suggestion
      const campaign = domainBuilder.buildCampaign({ organizationId: organization.id });
      const user = domainBuilder.buildUser({ memberships: [membership] });
```
Comme ça on rapproche l'organisation et la campagne. Là j'ai l'impression que c'est l'utilisateur qui crée la campagne

Reply:
Done